samples/subsys/fs/littlefs_blk/boards/nucleo_h743zi.conf

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2021 Lukasz Majewski, DENX Software Engineering GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# SDMMC1 connected SD card (via only 3 wires; CLK, MOSI, MISO)
```

Comment:
This seems to be a very individual solution, would not it be better to use a board or shield in the sample that already has a sdcard connector?

Reply:
Unfortunately, I cannot change the HW.

Reply:
> Unfortunately, I cannot change the HW.

OK, so together with the
> This example is supposed to work on :ref:`Nucleo H743ZI <nucleo_h743zi_board>`

you have actually tested that on that board? The board itself seem to be compatible with uno v3 (https://www.st.com/en/evaluation-tools/nucleo-h743zi.html), so shouldn't it allow shields like this https://store.arduino.cc/collections/shields/products/arduino-education-shield, with SD card connector, to work?

Reply:
@de-nordic I'm using custom "shield" PCB with simple connection of the SD card to work on this project.
The connection for the SDMMC card is as provided in the overlay file for nucleo and littlefs_blk sample program (in this PR).

Reply:
not addressed